### ISSUE TITLE
Feature request: Allow specification of co-authorship

### ISSUE THREAD

**Issue by abhi18av** [wjschne/apaquarto#100]
Hi @wjschne 👋 

Thank you so much for the wondeful template!

I was so excited to find this that I ended up spending quite a while and ported one of my manuscripts to the template already.

In the process, I realised a nice addition which is the ability to specify co-authorship in the metadata field. For now, the `equal` contribution doesn't translate as a `*` next to author names in the compiled outputs.

What do you think?

If I'm mistaken and this already exists, could you please point in the right direction?

**Comment by wjschne** [comment]
Good idea. Can you point me to an example of an APA document with this feature?

**Comment by wjschne** [comment]
The *Publication Manual* recommends that all agreements among authors about authorship be explained in paragraph 3 of the Author Note.  The apaquarto `author-note` has field called `disclosures`, which has a subfield called `authorship-agreements` where this kind of statement can go.

**Comment by abhi18av** [comment]
Thank you - makes sense 👍
